### PR TITLE
Bug 1221786 - Provide a way for users to set a usage_account_id

### PR DESCRIPTION
**Pull request by brenton**

### CODE REVIEW

**Comment by brenton** [comment]
[test]

**Comment by brenton** [comment]
@liggitt, I think @sferich888 mentioned this to you yesterday.  Is this all that's involved?

File: broker-util/oo-admin-ctl-user [hunk 1]
```diff
@@ -616,6 +641,13 @@ if args["--setplanquantity"]
   plan_quantity = args["--setplanquantity"].to_i
 end
 
+if args["--setusageaccountid"]
+  unless args["--setusageaccountid"] =~ /^[0-9]+$/
+    $stderr.puts "ERROR: Usage account ID must be a positive integer"
+    exit 1
+  end
+  usage_account_id = args["--setusageaccountid"].to_i
```
Comment (liggitt) [inline broker-util/oo-admin-ctl-user]: the type is actually a string in mongo, but looks like some code makes assumptions that it's an integer... for example, admin_helper.rb:
```
$user_hash.each { |k,v| billing_user_hash[k] = v if v['usage_account_id'] and (v['usage_account_id'].to_i > 0) }
```
requiring an int here is probably a good first pass

**Comment by liggitt** [comment]
@brenton, looks good as far as I know

**Comment by brenton** [comment]
[merge]

**Pull request merged**